CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ New functionality that are backwards-compatible:
 * [OLMIS-7649](https://openlmis.atlassian.net/browse/OLMIS-7649): Prepare administrator page for data export
 * [OLMIS-7705](https://openlmis.atlassian.net/browse/OLMIS-7705): Implement sending to backend 'data export'
 * [OLMIS-7707](https://openlmis.atlassian.net/browse/OLMIS-7707): Add the option of selecting export right + secure export page
+* [OLMIS-7673](https://openlmis.atlassian.net/browse/OLMIS-7673): Add import page
```

Comment:
We released the referencedata-ui module so this should not be under the released changes. The new header 5.6.10-SNAPSHOT / WIP should be added and a section title, then the ticket details. Similar as here: https://github.com/OpenLMIS/openlmis-referencedata/commit/dfff3b13ee05d7170b1d3310cf2ad0ade76d46ff